### DISCUSSION TITLE
How to change the design?

### DISCUSSION THREAD

**Discussion by Korenovskiy** [ra3xdh/qucs_s#689] [Q&A]
Hi everyone! I would like to redesing Qucs-s, but i can't get a bit how to do it. Guys, help me please. thanks in advance

**Comment by wawuwo** [comment]
"to redesign" is quite vague, what do you mean exactly?

**Comment by Korenovskiy** [comment]
I would like to add a new logo and add some windows with information, i need it for practise

**Comment by ra3xdh** [comment]
Application icons are inside `qucs/bitmaps/hicolor`. The *About Qucs-S* dialog is `dialogs/aboutdialog.cpp`

**Comment by zergud** [comment]
IMHO new logo and other graphics must be discussed before
You can attach image to this discussion
